adminsys_charter.md

```diff
@@ -0,0 +1,29 @@
+# YunoHost System Administrator Charter
+
+Because great powers imply great responsibilities, I commit myself as YunoHost adminsys to respect these points:
+
+## Security
+The reliability and security of our services is the responsibility of all, below are some rules to follow to avoid becoming an attack vector of the YunoHost infra :
```

Comment:
```suggestion
The reliability and security of the project's services is the responsibility of everybody. Below are some rules meant to prevent security breaches / leaks on the infrastructure:
```